package.json

```diff
@@ -147,6 +147,12 @@
         "schemes": [
           "element"
         ]
+      },
+      {
+        "name": "schildichat",
```

Comment:
This should go into `overlay-package.json`. We apply this on top of the normal `package.json` to ease merges: https://github.com/SchildiChat/schildichat-desktop/blob/sc/merge_helpers.sh#L218-L228